src/common/components/form/validationUtils.tsx

```diff
@@ -46,8 +49,30 @@ const validateDate = (value: string | number) => {
   }
 };
 
+/** validateyear()
```

Comment:
`validateyear()` in the comment is unnecessary and out-of-sync with the function name, how about just removing this?

Reply:
?

Reply:
It's a minor thing, just that the function is `validateYear(value: number)` and the comment said simply `validateyear()` so the name was off (i.e. validateYear is not validateyear, the case is off). What do you see as the value the duplication of the function name in the comment brings? I see it as unnecessary duplication myself because IDEs should show the real function name anyway and actually bringing negative value as it can be out-of-sync with the real function name and thus misleading.